keyboards/crkbd/keymaps/alairock/keymap.c

```diff
@@ -0,0 +1,262 @@
+#include "crkbd.h"
```

Comment:
This line should be changed to:

```suggestion
#include QMK_KEYBOARD_H
```

This will also automatically include `action_layer.h`, `action_util.h`, and `eeconfig.h`.